<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Weather</class>
 <widget class="QDialog" name="X2Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8006</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clarity II file setting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>454</x>
        <y>280</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>90</y>
        <width>300</width>
        <height>220</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>144</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ambient Temperature :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>156</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Humidity :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Point :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="ambientTemp">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="humidity">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>156</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-- %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="dewPoint">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>144</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sky Temperature :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="skyTemp">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>144</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sensor Temperature :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="sensorTemp">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-.- ºC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>144</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wind  speed :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="windSpeed">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>130</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- Km/h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>305</width>
        <height>160</height>
       </rect>
      </property>
      <property name="title">
       <string>Thresholds</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>32</y>
         <width>96</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SQM dark</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sqmThreshold">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>32</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="SQM">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--.- mpsas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>88</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current SQM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>281</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>The above thrershold is used to report dark condition to TheSkyX, if SQM is present in the Clarity II  file.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Clarity II file :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="filePath">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>520</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Check file</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileCheckStatus">
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>56</y>
        <width>240</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File check successful</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
